### PR TITLE
test: add a test to check the 'Create Volume' button is disabled in a ISCSI pool

### PR DESCRIPTION
**Pull request by yunmingyang**

### CODE REVIEW

**Comment by martinpitt** [comment]
This test [fails](https://logs.cockpit-project.org/logs/pull-13927-20200420-102833-880df459-ubuntu-stable/log.html#261-2) at the moment, but I also don't understand it very well. It feels like this may be simpler and more robust in the existing iSCSI tests, instead of creating a new pool just for this; and also doing the test in the `open()` helper function may not be the most straightforward place? @KKoukiou , can you please have a look at this? Thanks!

**Comment by martinpitt** [comment]
That said, thanks a lot @yunmingyang for looking at our verify tests! It's great that we can now collaborate on one set of tests that run everywhere!

**Comment by yunmingyang** [comment]
OK, I will modify and re-push, many thanks.

File: test/verify/check-machines-dbus [hunk 1]
```diff
@@ -2009,7 +2017,6 @@ class TestMachinesDBus(machineslib.TestMachines):
             remove=True,
         ).execute()
 
-
```
Comment (KKoukiou) [inline test/verify/check-machines-dbus:2012]: Unrelated whitespace change.

File: test/verify/check-machines-dbus [hunk 2]
```diff
@@ -1960,7 +1968,7 @@ class TestMachinesDBus(machineslib.TestMachines):
                 self.test_obj.assertEqual(size, int(self.size))
 
             def cleanup(self):
-                m.execute("virsh pool-destroy {0} && virsh pool-undefine {0}".format(self.name))
+                m.execute("virsh pool-destroy {0} && virsh pool-undefine {0}".format(self.pool_name))
```
Comment (KKoukiou) [inline test/verify/check-machines-dbus:1971]: This is correct fix but should be in separate commit since it has nothing to do with the new test.
When you repost with the new commit don't forget to add a Closes #13927 line in your commit message.

File: test/verify/machineslib.py [hunk 3]
```diff
@@ -283,7 +283,7 @@ def prepareStorageDeviceOnISCSI(self, target_iqn):
                   targetcli /iscsi/%(tgt)s/tpg1/acls create %(ini)s
                   """ % {"tgt": target_iqn, "ini": orig_iqn})
 
-        self.addCleanup(m.execute, "targetcli /backstores/ramdisk delete test && targetcli /iscsi delete %s" % target_iqn)
+        self.addCleanup(m.execute, "targetcli /backstores/ramdisk delete test && targetcli /iscsi delete %s && iscsiadm -m node -o delete" % target_iqn)
```
Comment (KKoukiou) [inline test/verify/machineslib.py:286]: This is not related to this commit. What are you trying to do though?Delete the target? looks like you need to specify the targetname for that as well.

Comment (yunmingyang) [reply test/verify/machineslib.py:286]: Oh, I just try to delete all records in the node mode of iscsiadm. Sometimes, I found that, when we try to create an ISCSI and the iqn is already in the 'iscsiadm -m node', then there will be an error raised even though there is no such session with the iqn.

File: test/verify/check-machines-dbus (hunk 1 above)
Comment (yunmingyang) [reply test/verify/check-machines-dbus]: Done

File: test/verify/check-machines-dbus (hunk 2 above)
Comment (yunmingyang) [reply test/verify/check-machines-dbus:1971]: OK, many thanks

**Comment by martinpitt** [comment]
[This failure](https://logs.cockpit-project.org/logs/pull-13927-20200427-094255-57673995-fedora-31-firefox/log.html#50-2) is a long-standing flake in testCreate(), so almost surely unrelated. Retrying.
